### DISCUSSION TITLE
17 alerts Frame rate under 20 fps on IVAO in 2 hours flights, average FPS 40

### DISCUSSION THREAD

**Discussion by lknfly1** [kubilus1/autoortho#148] [General]
Hi there, 

First it's a great thing that this talented author allows us to fly everywhere in the world with satellite imagery, a game changer for free!

Congrats and big thank you @kubilus1 

I'd like to know if there is a way to avoid freeze screen during the flight and especially during the approach. The issue that could kill the immersion occures from time to time, it's random, sometimes there no issue at all which is fantastic, other flights like the one I just did was very hard to handle, especially online IVAO ATC.  
The approach  on HECA (Cairo) was great, smooth in terms of perf 35, no stutter, even above very dense area, but once I touch down the screen froze and I have all of a sudden from 35 fps to 3 fps during the roling which killed my flight somehow!

If there is any trick around? I'll take it! 

Tks a lot Captains!

**Comment by lknfly1** [comment]
Unfortunately even with the last version it becomes unflyable online on my side because of huge stutters in cruise and particulary in approach which drops the fps from 40 to 5 fps today... that I had to cancel my flight twice in long final.

Using Bing as a source by the way, anyway I don't know what to do.

Most of the time it's ok with vfr GA low speed but once it gets to liners then it appears very difficult to get rid of stutters and frozen screen.s in a row..

**Comment by kubilus1** [comment]
A lot of time low frame rates around specific airports is an issue with
Ortho4XP, which this project relies on.  It may be worth asking on that
project as well: https://github.com/oscarpilote/Ortho4XP

**Reply by lknfly1** [reply]
Yeah but I've never experienced such stutters and frozen screens with the original orthox Z17 I have.
I just made a test with ARC instead of BING but it doesn't change, I could'nt even get to the cruise level from LFLU. 
No idea if it is because it loads the ortho scenery or else, but never had that before. 

I'll give a try with the orthoxp scenery from my SSD to see how it goes and I'll let you know. 
Tks for your answer ;-)

**Comment by loganlaporte** [comment]
I am getting the same issue on Vatsim/ x-pilot. I have even pulled the sliders way back. I think it has to do with data ingest and its all at once and not trickle, so its overwhelming everything. 
My pc is no slouch either. 7950x w/ 4090 suprim liquid, 64GB ddr5.

**Reply by lknfly1** [reply]
I didn't find a way to get rid of frozen screens (now flying from LFVP to CYYC) it is frozen since 6 min, I can't move anything and gotta wait like yesterday, same, even over less density area yet, I tried Process Lasso but nothing has changed, I have to try another different combination to use different cores... last try because unfortunately it becomes step by step almost unflyable on my side...
